### ISSUE TITLE
turtlebot/3ea2c30: liveness -> harmless?

### ISSUE THREAD

**Issue by gavanderhoorn** [robust-rosin/robust#415]
[turtlebot/3ea2c30](https://github.com/robust-rosin/robust/blob/master/turtlebot/3ea2c30/3ea2c30.bug) is about an env hook (basically a Bash script 'plugin') trying to find a directory which doesn't always exist (as it's a directory from a different package, which isn't listed as a dependency of the package in which the failing env hook is located):

https://github.com/robust-rosin/robust/blob/7e3932c4906a2851d1f84bfd96ecaa58924c9f57/turtlebot/3ea2c30/3ea2c30.bug#L4-L14

This is classified as `SOFTWARE:LIVENESS`.

However, the net effects of that env hook failing are essentially:

 1. an annoying warning on the console every time a user `source`s their `setup.bash` (ie: activates a workspace)
 2. the `TURTLEBOT_MAP_FILE` environment variable not getting set

but the base functionality of the package itself (ie: `turtlebot_bringup`) is not affected. The robot can still be controlled, no software will fail to work, *except* some other package which *might* use this variable (some `.launch` files in `turtlebot_navigation`).

Do we feel this is sufficient for a `LIVENESS` label?

---

Edit: and the env hook also only sets a "sane default".

**Comment by gavanderhoorn** [comment]
Yeah, I'm getting a bit confused about our definition of *liveness*. The following bugs all have `LIVENESS`:

```
confidential/2688e7a/2688e7a.bug
confidential/332f09f/332f09f.bug
confidential/c5dc9de/c5dc9de.bug
geometry2/12605ab/12605ab.bug
geometry2/15b2e3c/15b2e3c.bug
geometry2/245b02b/245b02b.bug
geometry2/566092b/566092b.bug
geometry2/7284788/7284788.bug
geometry2/74f0c66/74f0c66.bug
kobuki/03660af/03660af.bug
kobuki/054c753/054c753.bug
kobuki/38dce2a/38dce2a.bug
kobuki/3c4c399/3c4c399.bug
kobuki/6e748c1/6e748c1.bug
kobuki/acd5bfb/acd5bfb.bug
mavros/29af3a3/29af3a3.bug
mavros/86255ba/86255ba.bug
motoman/377d7be/377d7be.bug
ros_comm/eab0d3c/eab0d3c.bug
turtlebot/3ea2c30/3ea2c30.bug
universal_robot/bd1fce5/bd1fce5.bug
```

I've spot-checked these, and some don't describe runtime issues.

Could it be we've inflated liveness to include everything which could lead to loss of functionality?

**Comment by ipa-hsd** [comment]
All these bugs listed above seem to have `SYSTEM:LIVENESS` in the excel sheet. These are not `SOFTWARE:LIVENESS`
